### ISSUE TITLE
TODO: UnknownError

### ISSUE THREAD

**Issue by kabukabu1000** [taikoxyz/taiko-mono#15040]
### Describe the bug

Pressing the Transactions button displays TODO: UnknownErrore bug here.

### Steps to reproduce

Steps to reproduce here.

### Additional context

Additional context here.
![image](https://github.com/taikoxyz/taiko-mono/assets/91064362/ae5daaa9-df88-47c0-aa88-e084c0be9305)

**Comment by KorbinianK** [comment]
Thanks for the report @kabukabu1000 . Can you please reach out to me on Discord and provide the transaction details you try to claim? If you want, you can also paste the explorer link here and we'll look into it!

**Comment by kabukabu1000** [comment]
I bridged a few more times after that with no errors. The first transaction could be claimed after all, so the "TODO: UnknownError" may be a problem on the web server side.

I just compared the two screenshots and noticed that the ETH balance in the first image is zero. Perhaps the error occurred because could not get the wallet balance.

Here is the first transaction.
https://sepolia.etherscan.io/tx/0x2d7eff7660fe4ff13180ac22c68b2acaf4e5523aff3dd5f1d370352d084dbb1c

 ![image](https://github.com/taikoxyz/taiko-mono/assets/91064362/5699ef01-f60c-4ba4-b904-fedf88ea9cf8)
